### DISCUSSION TITLE
Where do you place class images?

### DISCUSSION THREAD

**Discussion by Portareumbra** [smy20011/dreambooth-gui#40] [Q&A] [answered]
In what location do I place the class images? The guide isn't clear

**Comment by futurevessel** [comment]
You don't have to create any class images, they will be generated by Stable Diffusion itself based upon the text you write in 'Class Prompt'.

If you want to use images you've handpicked, just enter the following parameter into the 'Training Arguments' box:
--class_data_dir=

and then add the path to the directory where you store your class images, on a *nix system it could be something like:
--class_data_dir=/mnt/data/class-images

In Windows:
-class_data_dir=C:\data\class-images

**Comment by smy20011** [comment, marked as answer]
It will generate image automatically by itself. If you want to use your own image, you can put it in the %APPDATA%/smy20011.dreambooth/{prompt}

**Reply by Portareumbra** [reply]
Is this the only way, or does the above work? Because futurevessel's method doesn't work

**Reply by smy20011** [reply]
Yeah, I think that's the only way to do it.

**Reply by Portareumbra** [reply]
Thank you

**Reply by futurevessel** [reply]
I can't be sure for Windows as I don't use it, but I am 100% sure that putting --class_data_dir=/path-to-your-class-images/ worked as I've done many tests with very specific class images and they certainly have a visible influence on the images generated by the trained model.
